### PR TITLE
added github.accessTokenVar configuration option

### PR DESCRIPTION
**Pull request by jack-kerouac**
This sets the username to the content of the environment variable specified in accessTokenVar and the password to empty.

### CODE REVIEW

File: tasks/grunt-release.js [hunk 1]
```diff
@@ -144,6 +144,20 @@ module.exports = function(grunt){
...
+      if((!!options.github.usernameVar) && (!!options.github.passwordVar)) {
+        username = process.env[options.github.usernameVar];
+        password = process.env[options.github.passwordVar];
+      }
+      else if(!!options.github.accessTokenVar) {
+        username = process.env[options.github.accessTokenVar];
+        password = '';
+      }
+      else {
+        throw 'either github.usernameVar and github.passwordVar or github.accessTokenVar must be set';
```
Comment (drublic) [inline tasks/grunt-release.js]: Please use a `grunt.fail` here.

**Comment by drublic** [comment]
Just this one request. Also, it would be perfect, if you could update the CHANGELOG.md with the changes you made.

Thank you so much!

**Comment by drublic** [comment]
Also please see #88.

**Comment by jack-kerouac** [comment]
done.

**Comment by drublic** [comment]
Great. Thank you so much for the work.

It would be very nice if you could try to add some tests for the behavior. If you're not into that, please let me know and we can have a look together.

**Comment by jack-kerouac** [comment]
I would have been happy to do that when opening the PR (which was more than four months ago). I moved on and I am working on something else now. I do not even use the plugin any more (using [grunt-github-releaser](https://github.com/dolbyzerr/grunt-github-releaser) now). So I will not do that any more. Sorry.

**Comment by drublic** [comment]
Thanks for the work. I've used your code as an inspiration for the implementation. It now landed in 111eaf4a5e6d4a305196e10c0d58e157de8abc68.

**Comment by jack-kerouac** [comment]
You are welcome! Glad to see it helped.

**Pull request closed without merging**